helm-charts/azure-api-management-gateway/values.yaml

```
@@ -14,6 +14,8 @@ nameOverride: ""
 fullnameOverride: ""
 
 gateway:
+  deployment:
+    terminationGracePeriodSeconds: 60
```

Let's not define a default
```suggestion
    terminationGracePeriodSeconds: 
```

---

terminationGracePeriodSeconds is set to 60 in the yaml that is generated from the portal. Shouldn't it be the same?

---

Not specifically, I think it's best to stick with Kubernetes defaults here